modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -1,62 +1,36 @@
 [[procedure-module-guidelines]]
 = Procedure Module Guidelines
 
-// [bhardest] - The original structure of the content here confused me since it's presented as a cross between a procedure template and a description of what a procedure is. I've taken a stab at reorganizing it in a way that hopefully is more clear and more consistent with template_concept.adoc.
+The required parts of a procedure module are a procedure and its introduction. Optionally, the module can also include prerequisites and additional resources.
 
-A procedure module is a procedure written with numbered steps -- what a customer needs to do to accomplish a goal successfully.
+.Schema of a procedure module
+image::procedure-diagram.png[]
 
-// [bhardest] - Here's a stab at a procedure diagram. Like with the concept, this can be made more professional later.
+[[writing-the-introduction]]
+.Writing the Introduction
+The introduction is a short description of the procedure. For example, it can be a lead-in sentence or an infinitive phrase (_To extract the certificate: <steps>_). See also _The IBM Style Guide_ footnoteref:[ibm-style-guide,DERESPINIS, Francis, Peter HAYWARD, Jana JENKINS, Amy LAIRD, Leslie McDONALD, Eric RADZINKSI. _The IBM style guide: conventions for writers and editors_. Upper Saddle River, NJ: IBM Press/Pearson, c2012. ISBN 0132101300.] for details on introducing procedures.
```

Comment:
I know :D Sorry, Monday might not be the best day for bad jokes.

But seriously, if a student of mine did this in an essay, I would have to tell them they need to add a list of all resources they used :)